### ISSUE TITLE
Webpage capture boefje is broken

### ISSUE THREAD

**Issue by praseodym** [minvws/nl-kat-coordination#1190]
The webpage capture boefje is broken on our testserver (https://kattest01.openkat.dev/en/mispoes/kat-alogus/plugins/boefje/webpage-capture/), we should investigate and fix any bugs we encounter. Might be related to #804.

#1191 would be helpful in debugging this.

**Comment by Darwinkel** [comment]
It's not only broken on the testserver. I can reproduce this locally.

**Comment by praseodym** [comment]
Container log:
```
2023-07-21T13:15:38.927497173Z npm WARN exec The following package was not found and will be installed: playwright@1.36.1
2023-07-21T13:15:39.678876631Z Executable doesn't exist at /ms-playwright/chromium-1071/chrome-linux/chrome
2023-07-21T13:15:39.678894423Z ╔══════════════════════════════════════════════════════════════════════╗
2023-07-21T13:15:39.678897423Z ║ Looks like Playwright Test or Playwright was just updated to 1.36.1. ║
2023-07-21T13:15:39.678899256Z ║ Please update docker image as well.                                  ║
2023-07-21T13:15:39.678901298Z ║ -  current: mcr.microsoft.com/playwright:v1.33.0-jammy               ║
2023-07-21T13:15:39.678902923Z ║ - required: mcr.microsoft.com/playwright:v1.36.1-jammy               ║
2023-07-21T13:15:39.678904548Z ║                                                                      ║
2023-07-21T13:15:39.678906340Z ║ <3 Playwright Team                                                   ║
2023-07-21T13:15:39.678908256Z ╚══════════════════════════════════════════════════════════════════════╝
2023-07-21T13:15:39.678910048Z     at Object.<anonymous> (/home/pwuser/.npm/_npx/e41f203b7505f1fb/node_modules/playwright/cli.js:17:18) {
2023-07-21T13:15:39.678913006Z   name: 'Error'
2023-07-21T13:15:39.678914590Z }
2023-07-21T13:15:39.682489173Z npm notice 
2023-07-21T13:15:39.682657965Z npm notice New minor version of npm available! 9.5.1 -> 9.8.1
2023-07-21T13:15:39.682663215Z npm notice Changelog: <https://github.com/npm/cli/releases/tag/v9.8.1>
2023-07-21T13:15:39.682665590Z npm notice Run `npm install -g npm@9.8.1` to update!
2023-07-21T13:15:39.682669423Z npm notice
```
Running `npx playwright` in the Docker container causes the NPM Playwright package to be installed (apparently it's not installed by default). It installs the latest version, i.e. newer than the version of Playwright in the Docker container, and when starting the NPM Playwright package throws an error because it is newer than the Playwright version in the container.
